packages/core/src/components/Rating/index.tsx

```diff
@@ -64,7 +64,7 @@ export class Rating extends React.Component<Props & WithStylesProps> {
               Star = IconStarFull;
             }
 
-            return <Star key={key} color={color} decorative />;
+            return <Star key={key} color={color} size="1.15em" decorative />;
```

Comment:
why this change? `Rating` already has size options that were based off of `1em`

Reply:
Slightly scaled up to match font size.